controller/inviteController.js

```diff
@@ -0,0 +1,121 @@
+const Invite = require('../model/inviteModel');
+const HttpStatus = require('http-status-codes');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+const create = (req, res) => {
+
+    const user = req.decoded;
+
+    //Only admins can create invite objects
+    if (!user || !ValidationUtil.isUserAdmin(user.accessType)) {
```

Comment:
No need to check if user is admin here with `ValidationUtil.isUserAdmin`. You can simply pass as the middleware authentification function the `auth.authAdmin` function which does it for you. You should use that for all functions/routes that require admin privileges per se.